.github/workflows/e2e-manual.yaml

```diff
@@ -0,0 +1,94 @@
+name: Manual E2E
+on:
+  # when https://github.com/community/community/discussions/11795 is resolved
+  # we will be able to dynamically build up the list of valid inputs.
+  # for now this needs to be manual.
+  workflow_dispatch:
+    inputs:
+      test-entry-point:
+        description: 'Test entry point'
+        required: true
+        type: choice
+        options:
+          - TestFeeMiddlewareTestSuite
+      chain-a-image:
+        description: 'The image to use for chain A'
+        required: true
+        type: string
+        default: "ghcr.io/cosmos/ibc-go-simd-e2e"
+      chain-a-tag:
+        description: 'The tag to use for chain A'
+        required: true
+        type: choice
+        default: main
+        options:
+          - main
+          - v4.0.0-rc3
+          - v3.0.0-rc2
+          - v2.2.0
+          - v2.1.0
+          - v2.0.0
+      chain-b-image:
+        description: 'The image to use for chain B'
+        required: true
+        type: string
+        default: "ghcr.io/cosmos/ibc-go-simd"
+      chain-b-tag:
+        default: v4.0.0-rc3
+        description: 'The tag to use for chain B'
+        required: true
+        type: choice
+        options:
+          - v4.0.0-rc3
+          - v3.0.0-rc2
```

Comment:
```suggestion
          - v3.0.0
```

Reply:
How do we get v3.1, v2.3 etc?

Reply:
to start with I just added a few basic entries. This are just images that exist already. All this list is doing is giving us a drop down menu. We can edit it as we see fit! We just need to make sure the images exist.